src/js/Rickshaw.Graph.js

```diff
@@ -69,6 +69,7 @@ Rickshaw.Graph = function(args) {
 	};
 
 	this._loadRenderers = function() {
+		this._bandRenderer = new Rickshaw.Graph.Band( { graph: self } );
```

Comment:
per the related question in Band.js,  it looks like line 78 would do this work for you if it's namespace to Renderer. With that said, I have not pulled the code down, so I'm not sure if that's possible.